things/postgres/channels.go

```diff
@@ -149,25 +155,29 @@ func (cr channelRepository) RetrieveAll(ctx context.Context, owner string, offse
 		items = append(items, ch)
 	}
 
-	cq := ""
-	if name != "" {
-		cq = `AND LOWER(name) LIKE $2`
-	}
-
-	q = fmt.Sprintf(`SELECT COUNT(*) FROM channels WHERE owner = $1 %s;`, cq)
-
-	total := uint64(0)
+	cq := `SELECT COUNT(*) FROM channels WHERE owner = :owner`
```

Comment:
I would do it similar like above since we alread have subqueries for metadata and name
cq := fmt.Sprintf(`SELECT COUNT(*) FROM channels WHERE owner = :owner %s%s`, nq, mq )
and no need for case at all

Reply:
You're right! I would even say that we don't need to do this checks since this name and metadata queries are returned by  `getNameQuery` and `getMetadatQuery`